app/models/cloud_volume.rb

```diff
@@ -9,15 +9,13 @@ class CloudVolume < ApplicationRecord
   belongs_to :availability_zone
   belongs_to :cloud_tenant
   belongs_to :base_snapshot, :class_name => 'CloudVolumeSnapshot', :foreign_key => :cloud_volume_snapshot_id
-  has_many   :cloud_volume_snapshots
-  has_many   :attachments, :class_name => 'Disk', :as => :backing
+  has_many :cloud_volume_snapshots
+  has_many :attachments, :class_name => 'Disk', :as => :backing
+  has_many :hardwares, :through => :attachments
+  has_many :vms, :through => :hardwares, :foreign_key => :vm_or_template_id
```

Comment:
Can you keep the symbol alignment with the belongs_to ones?  That will also prevent the first two lines from showing up as a diff.